### PR TITLE
support wildcard selector

### PR DESCRIPTION
**Pull request by jennyfel**
Create a wrapper for creation of dataviewWildcard selector required for conditionalFormatting support.
Expected usage inside CV code (enumerateObjectInstances) should be:

              import { dataViewWildcard } from "powerbi-visuals-utils-dataviewutils";
              ....
              public enumerateObjectInstances(options: EnumerateVisualObjectInstancesOptions): VisualObjectInstanceEnumeration 
              { 
                 ...             
                selector: 
             dataViewWildcard.createDataViewWildcardSelector(dataViewWildcard.DataViewWildcardMatchingOption.InstancesOnly),
               ....
              }

### CODE REVIEW

File: src/dataViewWildcard.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+*  THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+*  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+*  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+*  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+*  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+*  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+*  THE SOFTWARE.
+*/
+
+export const enum DataViewWildcardMatchingOption {
```
Comment (uve) [inline src/dataViewWildcard.ts:27]: should we add this enum to powerb-visual-api typings?

File: src/dataViewWildcard.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    /** Match Identities and Totals (default) */
+    InstancesAndTotals = 0,
+    /** Match Instances with Identities only */
+    InstancesOnly = 1,
+    /** Match Totals only */
+    TotalsOnly = 2,
+}
+
+export function createDataViewWildcardSelector(dataViewWildcardMatchingOption?: DataViewWildcardMatchingOption) {
+    if (dataViewWildcardMatchingOption == null)
```
Comment (uve) [inline src/dataViewWildcard.ts:37]: please add curly braces

**Review by uve — CHANGES_REQUESTED** [review]

File: src/dataViewWildcard.ts (hunk 1 above)
Comment (jennyfel) [reply src/dataViewWildcard.ts:27]: It's only required for below function so I don't see a reason to add it to visuals-api. I'm ok with either approach but I think single place is better

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+## 2.4.0
+* Add dataViewWildcard support
```
Comment (AviSander) [inline CHANGELOG.md:2]: 'for conditional formatting support' ?

**Review by AviSander — APPROVED** [review]

**Review by uve — APPROVED** [review]

**Pull request merged**